src/many-abci/src/module.rs

```diff
@@ -13,32 +12,44 @@ use many_types::blockchain::{
 };
 use many_types::{blockchain::RangeBlockQuery, SortOrder, Timestamp};
 use once_cell::sync::Lazy;
+use sha2::Digest;
 use std::ops::{Bound, RangeBounds};
 use tendermint::Time;
-use tendermint_rpc::{query::Query, Client};
+use tendermint_rpc::{query::Query, Client, Order};
 
 const MAXIMUM_BLOCK_COUNT: u64 = 100;
 static DEFAULT_BLOCK_LIST_QUERY: Lazy<Query> = Lazy::new(|| Query::gte("block.height", 0));
 
-fn _many_block_from_tendermint_block(block: tendermint::Block) -> Block {
+fn _many_block_from_tendermint_block<C: Client + Sync>(
+    block: tendermint::Block,
+    args: blockchain::ListArgs,
+    client: &C,
+) -> Result<Block, ManyError> {
+    let (count, order, query) = transform_list_args(args)?;
+    let transaction_results_by_id = tx_results(client, count, order, query)?;
```

Comment:
Since `args` is hardcoded, the `tx_results` parameters will always be the same

Reply:
@fmorency Not sure what you mean here . . . this is only hardcode in one of the two instances where it is used, and open to suggestions on what to set in the event that it is hardcoded, since introducing args in that place seems less natural